### PR TITLE
MTL-2355 Build aarch64 Variants of Fawkes for developers

### PR DESCRIPTION
**Pull request by jpdavis-prof**
### Summary and Scope

https://jira-pro.it.hpe.com:8443/browse/MTL-2355

- build fawkes artifacts for aarch64

#### Issue Type

- RFE Pull Request

### Prerequisites

- [x] I have included documentation in my PR (or it is not required)
- [x] I have tested this by building an image using this branch HEAD, or this is does not pertain to the node-images pipeline.
 
### Idempotency
yes
 
### Risks and Mitigations
None known.

### CODE REVIEW

File: group_vars/management_vm/packages.suse.x86_64.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+---
+packages_x86_64:
+  # rationale: Used for poking around the system
+  - conman=0.3.1-150500.1.2
+  - fawkes-httpboot-grub=1.0-1
```
Comment (rustydb) [inline group_vars/management_vm/packages.suse.x86_64.yml:28]: What is the reason for `fawkes-httpboot` moving into x86_64? The package is `noarch` https://github.com/Cray-HPE/fawkes-httpboot-grub/blob/main/fawkes-httpboot-grub.spec#L27

File: group_vars/management_vm/packages.suse.x86_64.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  - gru=0.0.4-1
```
Comment (rustydb) [inline group_vars/management_vm/packages.suse.x86_64.yml:29]: Same comment from `group_vars/fawkes_live/packages.suse.x86_64.yml`

File: group_vars/management_vm/packages.suse.x86_64.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  - metal-init=1.4.6-1
```
Comment (rustydb) [inline group_vars/management_vm/packages.suse.x86_64.yml:30]: This package is also `noarch`: https://github.com/Cray-HPE/metal-init/blob/main/Jenkinsfile.github#L44

File: roles/node_images_metal/files/scripts/metal/create-iso-artifact.sh [hunk 4]
```diff
@@ -33,6 +33,9 @@ ARCH="${ARCH:-}"
 if [ -z "${ARCH}" ]; then
     ARCH="$(uname -m)"
 fi
+if [ "${ARCH}" = "aarch64" ]; then
```
Comment (rustydb) [inline roles/node_images_metal/files/scripts/metal/create-iso-artifact.sh:36]:
```suggestion
elif [ "${ARCH}" = "aarch64" ]; then
```

**Review by rustydb — CHANGES_REQUESTED** [review]

File: roles/node_images_metal/files/scripts/metal/create-iso-artifact.sh (hunk 4 above)
Comment (jpdavis-prof) [reply roles/node_images_metal/files/scripts/metal/create-iso-artifact.sh]: That isn't the logic I'm looking for. We need to set ARCH if it's unset, then if ARCH is aarch64, change it to arm64.

**Review by rustydb — APPROVED** [review]

**Pull request merged**